SSRN.js

```diff
@@ -123,21 +124,15 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"notes": [],
-				"tags": [
-					"human rights",
-					"genocide",
-					"Japan",
-					"international law",
-					"nested analysis",
-					"mixed methods",
-					"event history analysis",
-					"survival models",
-					"hazard models",
-					"international norms",
-					"norm cascades"
-				],
-				"seeAlso": [],
+				"date": "2009",
+				"abstractNote": "What explains the large variation in the time taken by different countries to ratify the 1948 Genocide Convention? The costs of ratiﬁcation would appear to be relatively low, yet many countries have waited for years, and even decades, before ratifying this symbolically important treaty. This study employs a \"nested analysis\" that combines a large-n event history analysis with a detailed study of an important outlying case in order to explain the main sources of this variation. The initial event history history produces a puzzling ﬁnding: countries appear to be less likely to ratify the treaty if relevant peer countries have already done so. We use the case of Japan -- which has not yet ratiﬁed the Genocide Convention, despite the predictions of the event history model -- to explore the proposed causes of ratiﬁcation in more detail. Based on these ﬁndings, we suggest that once the norms embodied in a treaty take on a sufﬁciently \"taken-for-granted\" character, many countries decide that the costs of ratiﬁcation outweigh its marginal beneﬁts. The pattern of ratiﬁcation of the Genocide Convention therefore does not appear to ﬁt the classic model of the \"norm cascade\" that has been used to explain the adoption of other human rights norms. We conclude with suggestions for how the validity of our theory could be tested through a combination of further large-n and small-n analysis.",
+				"institution": "Social Science Research Network",
+				"libraryCatalog": "papers.ssrn.com",
+				"place": "Rochester, NY",
+				"reportNumber": "ID 1450387",
+				"reportType": "SSRN Scholarly Paper",
+				"shortTitle": "Who Doesn't Support the Genocide Convention?",
+				"url": "https://papers.ssrn.com/abstract=1450387",
 				"attachments": [
 					{
 						"title": "Full Text PDF",
```

Comment:
we should remove this, then -- otherwise it looks like a bug or something people should expect Zotero to do.